java/avro-converter/src/main/java/com/microsoft/AvroConverterConfig.java

```diff
@@ -0,0 +1,129 @@
+package com.microsoft;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.azure.core.credential.TokenCredential;
+
+public class AvroConverterConfig extends KafkaAvroSerializerConfig {
```

Comment:
Why not just have AvroConverterConfig extends from AbstractKafkaSerdeConfig? Where else do we use KafkaAvroSerializerConfig to warrant its creation here?